marklogic/assets/configuration/spec.yaml

```
@@ -0,0 +1,48 @@
+name: MarkLogic
+files:
+- name: marklogic.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
```

```suggestion
    - template: init_config/http
    - template: init_config/default
```
Also needs http options at init_config level